pkg/cmd/roachtest/tests/typeorm.go

```diff
@@ -164,14 +164,10 @@ func registerTypeORM(r registry.Registry) {
 			`cd /mnt/data1/typeorm/ && sudo npm test --unsafe-perm=true --allow-root`,
 		)
 		rawResultsStr := string(rawResults)
-		t.L().Printf("Test Results: %s", rawResultsStr)
 		if err != nil {
-			// Ignore the failure discussed in #38180 and in
-			// https://github.com/typeorm/typeorm/pull/4298.
-			// TODO(jordanlewis): remove this once the failure is resolved.
-			if t.IsBuildVersion("v19.2.0") &&
+			if strings.HasPrefix(cockroachVersion, "v21.2") &&
```

Comment:
do you know why only 21.2 is affected? i'd expect this to happen with any version?

Reply:
Yeah I think this should happen with any version.